ee/ucp/admin/configure/join-nodes/join-linux-nodes-to-cluster.md

```diff
@@ -135,3 +135,8 @@ Once you do that, you can start managing your UCP nodes:
 ```bash
 docker node ls
 ```
+
+## Use the API to manage your nodes
+You can also modify the UCP config through the [/api/ucp/config-toml API](/reference/ucp/3.1/api/).
```

Comment:
@david-yu what specific endpoints / HTTP methods can the end user work with for node management via the API? Since this page is under admin, I'm guessing only UCP administrators can hit these endpoints?

Can someone from engineering please add their input here?

Reply:
You can use either the node update API to add the orchestrator label (i.e. com.docker.ucp.orchestrator.kubernetes): /nodes/{id}/update

or the /api/ucp/config-toml API to change the default orchestrator setting: https://docs.docker.com/ee/ucp/admin/configure/ucp-configuration-file/#scheduling_configuration-table-optional 

cc @wsong

Reply:
@traci-morrison This comment needs to be added to the doc. The node update API docs should be part of the UCP API docs.

Reply:
@david-yu Can you approve this PR so that it can be merged? The previous writer already added the information to the doc:

https://deploy-preview-8723--docsdocker.netlify.com/ee/ucp/admin/configure/join-nodes/join-linux-nodes-to-cluster/